packages/components/src/local/organisms/planning-assets/index.tsx

```diff
@@ -21,7 +21,8 @@ export const PlanningAssets: React.FC<PropTypes> = ({ assets, forces, playerForc
       const lastSelectedAssetId = selectedAssets[selectedAssets.length - 1]
       const elmRow = document.getElementById(lastSelectedAssetId)
       if (elmRow && !preventScroll.current) {
-        elmRow.scrollIntoView()
```

Comment:
Hey @lebaphi  - how about this smooth scroll 😉

Reply:
sure @IanMayo , let me updated 😄

Reply:
Oh, the highlight is on the wrong row. I've implemented the smooth scroll:
![image](https://user-images.githubusercontent.com/1108513/199502813-27e55f75-3037-4f7c-8ef1-865240c7ed33.png)